scripts/bytecodecompare/storebytecode.sh

```diff
@@ -29,6 +29,12 @@ set -e
 
 REPO_ROOT="$(dirname "$0")"/../..
 
+if test -z "$1"; then
+	BUILD_DIR="emscripten_build"
```

Comment:
I would argue this file shouldn't default to `emscripten_build` as this file is not related to emscripten.

Reply:
Fair point. I misread the .travis.yml file where the call is.

Reply:
Probably also the reason it fails now